### PR TITLE
AssWalker Update + Mob Drops

### LINKED ISSUE #258: New mob idea for lavaland
A new NPC mob that I thought would be a cool addition to lavaland fauna.

**Marrow Weaver**
An oversized Sub-race of space Arachnids native to Lavaland it has Opal Black Chitin with glowing laval cracks running throughout its abdomen. It produces its own light source and is relatively easy to spot when not covered in Marrow Web.

**Weaver Durability**
Weavers themselves Are relatively easy to kill alone, should take 5 KA shots per weaver encouraging miners to finish them off as soon as possible and making them priority targets when encountered with other fauna.

**The Mobs AI and how it behaves**
It would be just about as big as a goliath, the catch is that it literally weaves webs of bone marrow which instantly harden. Instead of catching prey, its webs act as makeshift barricades that can be broken for bones. It will cast a certain amount of marrow webs around its general area wherever it goes, it will only aggro if its webs are attacked or run into. Upon being aggro'd it beelines quickly towards its target, does minimal brute damage but injects a venom that does steady stamina damage and if enough venom is injected it will stamina crit the target into unconsciousness. Once its prey is in stamina crit the Weaver sits ontop of them and will quickly weave them into a marrow cocoon. 

While in this cocoon you are still alive, albeit unconscious from the venom, if not removed from the cocoon after 5 minutes you will die and 3-6 weaver spawn will hatch from your corpse. Weaver spawn do similar damage to legion skulls but die in one hit, they are more aggressive than the adult Weavers but do not have any developed venom sacs. The stamina crit will likely only keep you down for half a minute giving you enough ample time to scream for help on radio in the hopes someone comes to rescue you, unless they fall victim to the Weaver/s

**A Different Spawner**
Could also do with an alternative tendril type, similar to the Ash Walkers have a Weaver Nest that is a tendril covered in small round egg clusters with strips of marrow web stretching off the tendril. Unlike normal tendrils the Weaver tendril can be exhausted of its spawns stopping it from spawning anything again, Maybe doesnt even drop a chest? (Could use discussion regarding that), but drops some special similar to how the ash walker nest drops an anomaly core.

**Butcher Remains**
1 Weaver Venom Sac
3 Sinew
1 Bone
5 Weaver Chitin

**Tribal additions**
Along with being a reliable source of bones for Ash Walkers the Venom sac could be applied to sharp weapons such as spears, axes, knives and daggers. Upon hitting a Carbon mob with the spear the first hit will deal a considerable amount of stamina damage along with the brute, the venom effect will then wear off on the weapon. Additionally it could cause the first hit against a fauna to deal double its normal damage, making the venom sac very valuable to Ash Walkers.

**Armor additions**

**Chitin Mail**
Weaver Chitin can be crafted into a jumpsuit type uniform from the tribal menu, it offers miner protection similar to that of the security jumpsuit. Crafted from 2 Weaver Chitin and 1 sinew, meaning 1 weaver supplies enough for 2 jumpsuits with an additional sinew.

**Chitin Gauntlets**
Crafted using 1 Weaver chitin and bone bracers, you reinforce the bone bracers with the chitin giving them more of an armor boost and they now cover your wrists AND your hands.

**Rigid Shield**
Made from 2 Weaver Chitin and 1 bone, blocks 3 melee attacks before breaking.

**Chitin Glaive**
Crafted from 1 Chitin and 2 sinew this weapon when thrown at a carbon mob has a high embed chance and does decent throw damage.

I did up a sprite to show what it would look like, I went with only 4 legs being visible because its bulkier than normal spiders so adding more than four causes a lot of the detail to get cut out from the legs overlapping the Spiders sprite.

**Marrow Weaver Sprites**

**Death Sprite**
http://puu.sh/pOU7V/ff2b54a5e5.png

**Venom Sac**

**Chitin mail**
Went with a less armored design, its not really mail, but it looks more tribal this way. Got the inspiration from the heart Protectors from kenshi.

**Chiten Gauntlets**
Made the gauntlets look really intimidating with the back spikes sticking out from the wrist.

### PR DESCRIPTION
**Pull request by Stealthkibbler**
Ash Walker Update + Mob Drops

The goal and purpose of this PR is to increase variety in ashwalker style of play, this includes some new craftable armors, fauna now drops more tribal materials and the nest has be en updated with some new stuff along with a brand new type of mob that spawns on startup.

**Craftables**
There are now 2 new sets of craftable armor, ashwalkers will also spawn with tatters instead of gladiator armor, they can also create a heart protector suit early on. There is also now a craftable deployable 3x3 sinew tent that can be made to create a permanent shelter from ash storms.

**The Pathfinder set**
Kasa- 2 bones, 2 sinew
Cloak- 6 sinew
Treads- 1 bone, 3 sinew
The Pathfinder gear gives you protection against ash storms however it has almost little to no armor against any other forms of damage.

**The Chitin Set**
Plate Armor- 5 bones, 2 chitin, 3 sinew
Gauntlets- 3 bones, 1 chitin, 1 sinew
Boots- 1 bone, 1 chitin, 1 sinew
The Chitin set is for true warriors, bolstering with just as much armor as riot gear it will make the difference in a fight when confronting the deadly fauna of lavaland

**New Spawn tatters**
Being spawned from a giant hellish tendril egg isn't exactly the most hygenic of ways to come into the world, but on the bright side atleast the remnants of your embryo will look the part on your scales.

**Heart Protector**
1 bone, 1 sinew
Crafted from 1 bone and 1 sinew this nifty piece of armor offers a small amount of protection, the upside is that it can be worn under armor since it acts as a uniform.

Sprites from left to right
Tattered sinew, Heart Protector, Chitin Set, Pathfinder Set.

**Sinew Tent**
You can now craft a deployable 3x3 sinew tent, this object can be made from 3 sinew, 2 bone and 6 sandstone. There is no actual tent sprite but it is noticeable by its cult engraved flooring and sandstone centre.

**Marrow Weaver**
The Marrow Weaver is now a new mob that will spawn in lavaland, a menacing mutated version of the space arachnid it runs fast, chases targets for long distances and spins webbing all over the caverns of lavaland, beware its deadly venom.

**Mob Drops**
Fauna will now drop more materials than they use to, not only is this to compensate for the new craftables but also because it was a rarity to actually see an ashwalker with the default bone armor. Bone, Sinew and Chitin are now being treated as very common crafting materials and therefore you will find good amounts of them in just 1 mob alone bar the chitin.

Goliath now drops 5 bones instead of 2
Watcher now drops 3 Sinew and 2 Bones instead of 2 sinew and 1 bone.
The Marrow Weaver drops 7 bones, 3 sinew and 6 chitin.

These drop amounts sound like a lot, however  this guarantees that killing just 1 fauna will reward a tribe. It was very counterproductive to spend 1-4 minutes killing a goliath for just 2 bones, likewise watchers pretty much guaranteed medical supplies for their measly 1 bone and 2 sinew. Legions will retain their purpose as a farmable temporary instant heal and their drops have not been changed

**Ash Walker Nest**
The nest now has an infinite water source so that ash walkers can spend storms washing their used bandages and whatnot. It is located in a small sandy room to the right of the tendril.

fixes #258

### CODE REVIEW

**Comment by Stealthkibbler** [comment]
@KorPhaeron this is the PR I was talking about

**Comment by ShadowDeath6** [comment]
making full bone armor with 2 goliaths is dumb, make it like 3 bones, not 5

**Comment by Stealthkibbler** [comment]
@ShadowDeath6 No.

5 bones is fine for spending 1-4 minutes killing a goliath provided you dont get injured in the process, it still offsets the amount needed and also rewards you with some spares provided you manage to kill a second one.

Players shouldn't be starved of resources, they should be amply rewarded with just enough to progress for tackling fauna. Lower the values too much and it just becomes too tedious since you spend way too much time trying to kill enough mobs just to get 1 item turning it into a grind. Players shouldn't have their minds set on one 1 piece of armor for any longer than they need to, the whole idea of even having to kill more than 2 goliaths for 1 piece of armor is stupid in of itself. 

Besides that you probably have never heard of everything on lavaland has armor penetration, because newsflash Shadow, it does, and 40 of it too. 

And if they find a tendril and farm it..so what? RNG blessed them with the chance to farm for resources and if they want to spend the time and effort doing so then good for them, that was their choice as players to work together to get the gear that they wanted.

Mob difficulty stays the same, however now they actually give you decent rewards.

**Comment by ShadowDeath6** [comment]
I don't know about you, but if a miner shoots a goliath about 6 times with their accelerator, they get a kill. That's not hard. 
Lavaland isn't ashwalker simulator, it's a mine with added benefits and ghost roles.
I get that you like ashwalkers, but giving miners, the main people who are supposed to be in the mine, stupidly easy to get armor upgrades makes it much easier to stomp all fauna you come across bar things like tendril ganking and megafauna.
Not to mention ashwalkers being able to make multiple bone axes in a few kills, which makes their lives extremely easy since they can use the axe to give themselves infinite healing via legion souls.

tl;dr H U G B O X, but the new mob is pretty cool and the sprites are good

**Comment by GrayRachnid** [comment]
Agreed with shadow, and it doesn't take 1-4 minutes to kill a goliath, it took me approximately 30 seconds to kill a goliath as an ass walker with a spear and no armor.

**Comment by Stealthkibbler** [comment]
Miners rarely ever use the tribal crafts, most the time they go for bigger goals like killing tendrils, also the miners explorers suit is still extremely strong when upgraded with goliath plates. If a miner also goes for the chitin plate they lose their temp resists, likewise if a miner makes the pathfinders set they lose all their armor for survivability during ash storms, even if they tried to carry everything with them to swap out equipment, thats gonna cost them a lot of space which is already chewed up by all their other equipment. Miners will have to sacrifice inventory and gear to use the tribal stuff regardless of how easy it is for them to kill stuff. 

That being said everything is already easy for miners to kill, its all melee, they simply have to kite, the drops on the Weaver are so high because it runs at sprinting speed meaning you have to risk being injected with venom which bypasses any layers of armor you might have, get bit too much without charcoal or anti-tox and you're as good as dead not to mention the webs are now going to prevent miners from lolrunning through caves since they can get caught in them.

As for Ashwalkers already being able to make multiple bone axes, the most I've seen in 1hr+ rounds is 2 bone axes and 1 bone spear, Ash walkers have to sacrifice way too much when it comes to deciding what to craft.

Just by looking at the damage values you realise that they are very small upgrades from eachother, its arguably better to get a spear over an axe since mob drops are so so low.
dagger does 15 dmg
spear does 20
axe does 23
the +3 damage on the axe just isnt worth the time and effort it takes to get it, especially with the bone spear only being like 1-2 damage upgrade from the normal improv spear that they spawn with. However most Ash Walker players dont understand the damage difference, they just see that it costs more and assume its way better, when realistically the weapons barely make a difference in any fight.

I talked to Kor Phearon and he agreed that the crafting recipes are way too expensive for what they offer, so instead of making the crafting recipes cost less, I've made the mobs drop more, because logically the more a mob drops the more rewarding it is. You still retain that extra challenge from fighting the mobs given that they stay the same but atleast you wont have to spend the whole fucking round hoarding your own mats just to get 1 item for yourself.

Additionally its meant to encourage teamplay among ashwalkers, so many ashwalkers hoard materials for themselves because they can never get around to acquiring what they want, it eventually gets to a point where nobody bothers making armor because logically if you can kill something before it hits you then you dont need armor in the first place. If I'm an ashwalker and I kill a goliath, I want those bones for myself until I can get my armor, the sooner I get my armor the sooner I can share my spare materials with other ashwalkers.

Consider this, if a goliath drops 3 bones, that's 2 goliaths per ashwalker, most long rounds with ashwalkers tend to end with around 10-30 ashwalkers, that's 20-60 goliaths needed to be killed, world gen tends to sparsely spawn every mob atleast 12 tiles away from eachother, so consider that every 12 tiles a goliath may or may not spawn outside of a legion, weaver or watcher.  This means that just to get a single bone armor for every walker, the tribe needs to cover almost all the ground in lavaland. That is very shit.

However if a goliath drops 5 bones then that means every 3 goliaths 2 walkers can have armor with some spare leftover. This is way more productive than having it be 3.

This of-course is ignoring tendril spawn, tendrils are godsends to ash walkers anyway and remain the same regardless of what they drop.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -93,6 +93,28 @@
...
+	armor = list(melee = 0, bullet = 0, laser = 25, energy = 0, bomb = 0, bio = 0, rad = 0)
+
+/obj/item/clothing/shoes/chitentreads
+	name = "Chiten Boots"
+	desc = "Compact boots crafted from a weavers chiten with interlacing sinew"
+	icon_state = "chitentreads"
+	item_state = "chitentreads"
+	strip_delay = 50
+	put_on_delay = 50
+	armor = list(melee = 35, bullet = 35, laser = 0, energy = 10, bomb = 25, bio = 0, rad = 0)
```
Comment (Notamaniac) [inline code/modules/clothing/shoes/miscellaneous.dm:119]: Aren't these armour values redundant since there aren't any flags indicating where it covers? Or am I being codeblind again?

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -136,13 +136,33 @@
...
+obj/item/clothing/under/tatters
+	name = "Sinew Tatters"
+	desc = "The remnants of an Ash Walkers birth, still sticky and fleshy to the touch"
+	icon_state = "tatters"
+	item_state = "tatters"
+	fitted = NO_FEMALE_UNIFORM
+	can_adjust = 0
+	body_parts_covered = CHEST|GROIN|LEGS
+	armor = list (melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+
```
Comment (Notamaniac) [inline code/modules/clothing/under/miscellaneous.dm:164]: What's the point of the armour var here if they're all set to zero?

Comment (Notamaniac) [inline code/modules/clothing/under/miscellaneous.dm]: What's the point of the armour var here if they're all set to zero?

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 1 above)
Comment (Stealthkibbler) [inline code/modules/clothing/shoes/miscellaneous.dm:119]: All other shoes dont state which parts they protect, I assume this is because shoes can only protect the legs so it defaults to that anyway. I'll change it to cover legs anyway and compile to see if it recognizes the verb.

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (Stealthkibbler) [inline code/modules/clothing/under/miscellaneous.dm:164]: A leftover from me copy/pasting, is getting removed.

**Comment by Notamaniac** [comment]
With a fully upgraded explorer suit, the heart protector gets you 85% brute resistance and 40% bullet resist for the chest. The weaver chitins let you get 95%  brute resist(reduced to 90% due to armour cap)  ,70% bullet resist and 65% laser resist for the arms. For the legs, you can get to 90% brute resist cap, but you'll only get 55% bullet resist and you're stuck with the default 20% laser resist.

I'm not entirely sure if this is OP or not, since most traitor weapons have AP values to get through armour, and most combat is reliant on stuns rather than taking damage. Not to mention that you can aim for the head which is the least protected. 

I'm ambivalent to the PR, but I still think that the high values that you can reach need considering.

Edit : You've also triggered me by using pure black on the weaver sprite.

**Comment by TheSadFox** [comment]
You also can get 125% for legs with chitin boots, armor and heart protector. But yeah, don't think its gonna be an issue in a PvP combat, where you will die to one stun anyways, no matter the armor.

> armour cap

Won't values higher than 90% still affect weapons with penetration tho?

**Comment by imtakingabreakdontatme** [comment]
What's wrong with your bots?

**Comment by Notamaniac** [comment]
> Won't values higher than 90% still affect weapons with penetration tho?

Yup, looking at AP code, it looks like the AP is subtracted first, then then 90% armour cap is applied

**Comment by Stealthkibbler** [comment]
@GrayRachnid 
@ShadowDeath6 

The armor values were just something I put in as a standing point, definitely subject to change based on what people think, but you have to also consider that everything on lavaland has 40 armor penetration. Meaning 40% of that armor value is not doing anything for you. It could potentially become a bit overpowered if a miner brought it onto the station given its values, however it's not as round impacting than a miner bringing a tendril super artifact onboard to interfere with the antagonists, I've fucked with a space ninja before using the angel wings, he got so frustrated with how fast I was that he simply detonated himself. Lavaland itself just doesnt fit station life, its meant for lavaland, the only way to remedy that is to make lavaland stuff somehow null on the station, but then you'd summon an unholy lynch mob from the depths of hell and god forbid something like that even be mentioned on TG without having your throat stuffed with the salt of 1000 lavaland elitists.

I'm fine with people giving me values that they think is fair and adjusting them to meet that comparable standard. In the end I'm likely to make it so you need bone armor to even get the chitin plate so there's a sense of climbing progression with upgrading your armor, the plate is meant to give a lot of brute resist but it gives shitall anything else, meaning a cabbage person or human wearing it would still get fried all the same from lasers than not wearing it. The protector might be a bit too high, I was planning on just making it an easy grab version of the sec jumpsuit, I mean atm it has just as much value as bone armor which is a bit questionable, will probably lower it to 10-15 brute and bullet protection. As for Pathfinder it works as intended, gives some laser resist but gives absolutely 0 protection against anything else. 

@Notamaniac 
I used the black and red because I based the design off the Australian Black Widow spider, most venomous spiders usually have a very alerting or brightish abdomen marking to signify that they are deadly without the abdomen black and red it would literally look like a giant moving rock with eyes.

@KorPhaeron What do you mean?

**Comment by oisin100** [comment]
> What's wrong with your bots?

Weird, Shit. Ill fix it... At some point...

**Comment by imtakingabreakdontatme** [comment]
Would be appreciated, it's a nightmare to scroll through on mobile

**Comment by Super3222** [comment]
For craftable armor the chintin armor is a bit overpowered. Now, when you're making craftable weapons and items, you have to take into account that these things can be multiplied up to 10x. With the ingredients required in the craft, any ashwanker would definitely deserve powerful armor, but the armor values are a little much.

armor = list (melee = 65, bullet = 35, laser = 15, energy = 10, bomb = 35, bio = 0, rad = 0)
I honestly think that the only problem with them is the melee. I mean protecting 65% is incredibly powerful, just look at the captains space suit.

> armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50)

from https://github.com/yogstation13/yogstation/blob/9ca634e7a20196faa263bda7fc5a35f687adab92/code/modules/clothing/spacesuits/miscellaneous.dm#L34

That would mean that the armor's stats are somewhere inbetween the captains space suit (which is argubably the most toughest armor avaliable during a normal round) and the DeathSquad's armor.

I think lowering it so that it matches the captains space suit in protection would be fair.

One more thing, I don't think armor for gloves and boots even exists anymore. Unless you look at the Space Ninja code, but I'm almost certain that feet/hands aren't recognized when it comes to armor. Therefore, glove and boot armor wouldn't present any actual armor values. I also think it'd be a bad idea, since with the current values for them it gives potential to stack up very high.

Also like we mentioned before in #coder, I think that there should be a few 5-6 pieces of paper in the ashwalker nest that are required for you to build all of these things. Therefore, if miners want to craft things then they don't get the knowledge from the top of their mind. This'd also facilitate the vast majority of unique tribal stuff with the actual tribe. So if they don't exist, nothing too tribey (like the items here which are good) goes into the wrong hands.

It'd basically be a wrinkled up, old, and sort of burnt piece of paper. It wouldn't do anything special, but just require you to hold it while crafting tribe accessories.

**Comment by imtakingabreakdontatme** [comment]
> One more thing, I don't think armor for gloves and boots even exists anymore. 

It actually stacks on arms/legs now, which leads to weird situations where armoured boots + riot armour = invincible legs

**Comment by Stealthkibbler** [comment]
> For craftable armor the chintin armor is a bit overpowered. Now, when you're making craftable weapons and items, you have to take into account that these things can be multiplied up to 10x. With the ingredients required in the craft, any ashwanker would definitely deserve powerful armor, but the armor values are a little much.
> 
> armor = list (melee = 65, bullet = 35, laser = 15, energy = 10, bomb = 35, bio = 0, rad = 0)
> I honestly think that the only problem with them is the melee. I mean protecting 65% is incredibly powerful, just look at the captains space suit.
> 
> armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50)
> from https://github.com/yogstation13/yogstation/blob/9ca634e7a20196faa263bda7fc5a35f687adab92/code/modules/clothing/spacesuits/miscellaneous.dm#L34
> 
> That would mean that the armor's stats are somewhere inbetween the captains space suit (which is argubably the most toughest armor avaliable during a normal round) and the DeathSquad's armor.
> 
> I think lowering it so that it matches the captains space suit in protection would be fair.
> 
> One more thing, I don't think armor for gloves and boots even exists anymore. Unless you look at the Space Ninja code, but I'm almost certain that feet/hands aren't recognized when it comes to armor. Therefore, glove and boot armor wouldn't present any actual armor values. I also think it'd be a bad idea, since with the current values for them it gives potential to stack up very high.
> 
> Also like we mentioned before in #coder, I think that there should be a few 5-6 pieces of paper in the ashwalker nest that are required for you to build all of these things. Therefore, if miners want to craft things then they don't get the knowledge from the top of their mind. This'd also facilitate the vast majority of unique tribal stuff with the actual tribe. So if they don't exist, nothing too tribey (like the items here which are good) goes into the wrong hands.
> 
> It'd basically be a wrinkled up, old, and sort of burnt piece of paper. It wouldn't do anything special, but just require you to hold it while crafting tribe accessories.

I'm going to assume you didnt read the recent commits, ashwalkers now spawn with 150 stacks of mythril, this mythril is used to make all the armor after normal bone. If a miner wants to make it they can raid the nest and take the mythril.

how many times do I have to point out that lots of armor doesnt make you invincible on lavaland, every mob has 40 armor penetration meaning only a portion of the actual armor is being accounted for.

The default armor of the mining suits melee prot is 30, which can be upgaded three times, if that isnt enough for you miners have ranged weaponry, meaning that in the event they get caught in melee they have a bunch of armor to mulligan their asses out of the situation. Walkers dont have ranged, at all, they HAVE to rely on melee, they need that protection and even with it only a portion of it is accounted for when being actually hit.

And if they bring it on the station? What can armor not stop? Tazers, Flashbangs, Baton stuns, revolver bullets, disabler beams, flashes, restraints, stunprods, the clowns PDA, Banana peels, wet floor, shocked grilles. There's so many ways to deal with an armored target that you just look at the armor values and think "That's OP, he's almost unkillable" When all it takes is one stun and that person is fucked. 

The station doesnt offer as much robust armor options because not everything on the station has armor penetration, the station isnt a kill or be killed environment, the station has a functional security force with a way to disable threats in multiple non lethal manners. None of this matters on lavaland, lavaland is word for word, the Hell Death Fuck Land, its a kill or be killed environment, you have no hard stun options against any of the hostilities and there is no security force or chain of command to protect you. It's survival of the fittest, you kill or you be killed, the only friends are your fellow faction members, even golems have a tendency to backstab miners if they do something that aggravates them.

**Comment by Stealthkibbler** [comment]
MORE FEEDBACK PLS, I've been going a bit hard on the recent feedback but I still encourage people to give their opinions on the matter especially the recent commits.

**Comment by Super3222** [comment]
We should give this PR a go, and see how it does in the game. I'm not a fan of the 5 bones for one goliath but, we'll just have to see if it's too much from the results.

**Comment by FluffySurvivor** [comment]
:+1:

**Comment by Stealthkibbler** [comment]
Doesnt seem like this will be getting anymore feedback.

**Comment by Stealthkibbler** [comment]
@Iamgoofball tactical goofball can u gibe me ur criticism.

**Comment by TheSadFox** [comment]
Why.

**Comment by Cruix** [comment]
@Stealthkibbler If you do not want me to merge any of your stuff, just close the PR one more time and I will not touch it again.

**Comment by Iamgoofball** [comment]
ABUSE

**Comment by Stealthkibbler** [comment]
I'll fix the conflicts @Cruix Like I stated in coderchat before council pulled a stalin on me I'm happy to help with my recent PRs if people want them merged. I'm not going to throw tons of potential content for people to enjoy down the drain over a dumb feud.

**Comment by Stealthkibbler** [comment]
conflicts fixed, if you wanna merge it, merge it before something else conflicts or some shit.

**Comment by Stealthkibbler** [comment]
fuck you travis CI

Well I dunno what the fucks wrong with it, it was fine before some other stuff got merged so who knows whats dicking with travis

**Comment by Cruix** [comment]
#193 moved a bunch of stuff from lavaland_ruins_code.dm to some other files.

**Comment by X-TheDark** [comment]
Fucking hell, I'll see if I can tell what exactly needs to be changed here so it doesn't fail, after I'm done with fixing antag weights.

**Comment by Stealthkibbler** [comment]
Goddamit @X-TheDark ya just had to go and move everything, anyhow there's not much I can do since I dont know the details of all of it, I could zip up all my changed files and give it to you or cruix, as long as I dont rename the base files where they belong its pretty self explanatory.

**Comment by X-TheDark** [comment]
I'll see if I can re-port this over to the new code changes.

**Comment by Cruix** [comment]
If you can figure out how, you can rebase this branch to an earlier commit and accept my pull request. Otherwise, I can just merge the branch I made.
